### ISSUE TITLE
fix(modal): shift+tab without focused element within modal

### ISSUE THREAD

**Issue by Hardhik** [twbs/bootstrap#21661]
Issue can be validated here: 
1) Navigate to http://getbootstrap.com/javascript/ 
2) Search for "Launch demo modal" and click on that. Now use shift+tab, nothing happens.

There was a similar bug in Angular-ui's bootstrap which got fixed: https://github.com/angular-ui/bootstrap/pull/5294

This should be done even in this library too.

I have tried on:
Operating system and version Windows, 
Browser and version Chrome: 55.0.2883.87

**Comment by patrickhlauke** [comment]
Not going to get any fixes for v3, but relabelled as v4 as it happens there as well

**Comment by patrickhlauke** [comment]
closing this issue here - tracking the problem in the meta-issue https://github.com/twbs/bootstrap/issues/28481
